config/smarty.config.inc.php

```diff
@@ -175,8 +175,9 @@ function smartyCleanHtml($data)
     }
 }
 
-function smartyClassname($classname)
+function smartyClassname(string $classname)
 {
+    
```

Comment:
```suggestion
```
no need for this empty line 👍🏻 other than that, lgtm 👍🏻

Reply:
Done